src/constants/index.ts

```diff
@@ -26,4 +26,6 @@ const diets = [
 const ingredientImageUrl = (imageName: string) =>
   `https://img.spoonacular.com/ingredients_500x500/${imageName}`;
 
-export { colorSchemes, cuisines, diets, ingredientImageUrl };
+const FAVOURITES_RECIPES = 'favourites';
```

Comment:
what is this?

Reply:
it's storage key constant to use with `async storage` for saving favorite recipes in local storage

Reply:
define all storage keys in single a object